Dockerfile

```diff
@@ -1,11 +1,26 @@
-FROM ghcr.io/comocheng/kineticmodelssite/kms-build:latest
+# Fetch RMG-Py build
+FROM comocheng/kms-rmgpy:latest as rmgpy
 
-COPY environment.yml /app/
-RUN conda config --set channel_priority strict
-RUN conda env update --prefix $ENV_PREFIX --file environment.yml
-RUN conda clean -afy
-ENV PATH /kms_env/bin:$PATH
-ENV CONDA_DEFAULT_ENV /kms_env
-RUN source activate /kms_env
+# Application
+FROM python:3.9-alpine
 
-COPY . /app/
+ENV PYTHONUNBUFFERED=1 \
+  PYTHONDONTWRITEBYTECODE=1 \
+  POETRY_HOME="/opt/poetry" \
+  POETRY_NO_INTERACTION=1
+
+COPY --from=rmgpy /rmg/RMG-Py/rmgpy /rmgpy
+
+RUN apk add --no-cache curl \
+  && (curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -)
+ENV PATH="$POETRY_HOME/bin:$PATH" \
+  PYTHONPATH="/rmgpy:$PYTHONPATH"
```

Comment:
Note that RMG-Py is not specified as a dependency in the `pyproject.toml` - the dependency is added to the PYTHONPATH here